dotnet-datetime/GetDatePartFromDateTime/GetDatePartFromDateTime.Tests/DateTests.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace GetDatePartFromDateTime.Tests
+{
+    [TestClass]
+    public class DateTests
+    {
+        [TestMethod]
+        public void RandomDate_GetDateProperty_DateComponentsAreEqual()
+        {
+            DateTime fullDate = new DateTime(2022, 02, 14, 10, 40, 00);
+
+            DateTime datePart = fullDate.Date;
+
+            Assert.IsTrue(datePart.Year == fullDate.Year && datePart.Month == fullDate.Month && datePart.Day == fullDate.Day);
+        }
```

Comment:
Please check the guidelines for the proper test naming convention and change all the method names.